src/Microsoft.AspNet.Mvc.ApiExplorer/project.json

```diff
@@ -0,0 +1,19 @@
+﻿{
+  "description": "The API explorer component for ASP.NET MVC.",
```

Comment:
Like, saying what an API description is... something about discovering metadata about the controllers, actions, or whatever.